blivet/devicelibs/dasd.py

```diff
@@ -298,15 +298,18 @@ def write_dasd_conf(disks, root):
     # this does catch every DASD, even non-aliases, but we're only going to be
     # checking for a very specific flag, so there won't be any duplicate entries
     # in dasd.conf
-    devs = [d for d in os.listdir(sysfs) if d.startswith("0.0")]
-    with open(os.path.realpath(root + "/etc/dasd.conf"), "a") as f:
-        for d in devs:
-            aliasfile = "%s/%s/alias" % (sysfs, d)
-            with open(aliasfile, "r") as falias:
-                alias = falias.read().strip()
-
-            # if alias == 1, then the device is an alias; otherwise it is a
-            # normal dasd (alias == 0) and we can skip it, since it will have
-            # been added to dasd.conf in the above block of code
-            if alias == "1":
-                f.write("%s\n" % d)
+    try:
+        devs = [d for d in os.listdir(sysfs) if d.startswith("0.0")]
+        with open(os.path.realpath(root + "/etc/dasd.conf"), "a") as f:
+            for d in devs:
+                aliasfile = "%s/%s/alias" % (sysfs, d)
+                with open(aliasfile, "r") as falias:
+                    alias = falias.read().strip()
+
+                # if alias == 1, then the device is an alias; otherwise it is a
+                # normal dasd (alias == 0) and we can skip it, since it will have
+                # been added to dasd.conf in the above block of code
+                if alias == "1":
+                    f.write("%s\n" % d)
+    except OSError:
+        pass
```

Comment:
The config file (/etc/dasd.conf)? No. It's the sysfs directory (/sys/bus/ccw/drivers/dasd-eckd) that doesn't exist and causes a traceback. This happens if you install only with FBA DASDs -- they do not use the ECKD driver, hence why the sysfs dir being checked doesn't exist (I believe it would be /sys/bus/ccw/drivers/dasd-fba for FBA DASDs).

FBA DASDs are pretty uncommon nowadays; most DASDs now are of the ECKD variety. If you install with a mixed group of DASDs (ECKD and FBA) this problem doesn't occur (since the existence of ECKD DASDs means /sys/bus/ccw/drivers/dasd-eckd will exist). And AFAIK if you have one of these HyperPAV aliases, it's not going to ever show up as an FBA disk.

I'll add a comment to explain this further (I didn't realize it was unclear what the issue is; oops :-/), and I can log the error for informational purposes, but I don't think any real issue is being avoided.